bookwyrm/utils/isni.py

```diff
@@ -0,0 +1,152 @@
+"""ISNI author checking utilities"""
+import xml.etree.ElementTree as ET
+import requests
+
+
+def url_stringify(string):
+    """replace spaces for url encoding"""
+
+    # TODO: this is very lazy and incomplete
+    return string.replace(" ", "+")
```

Comment:
Is `urllib.parse.quote_plus` the ticket here? (https://docs.python.org/3/library/urllib.parse.html#urllib.parse.quote_plus)